web3.js/src/message/legacy.ts

```diff
@@ -51,6 +72,28 @@ export class Message {
     );
   }
 
+  get version(): TransactionVersion {
```

Comment:
I personally feel that it's valid to talk about the version of a transaction when you really mean the version of a transaction's message. The only thing that makes this a "message version" to me is the need to encode the version in the signed message, but it really refers to the structure of the transaction as a whole. It's likely that in the future, the structure of the signatures portion of the transaction is updated and then versioning won't be specific to the message structure as it appears to be right now since v0 transactions didn't update how signatures are encoded.